.travis.yml

```diff
@@ -0,0 +1,24 @@
+
+dist: bionic
```

Comment:
Is there a reason for going with a particular release of Ubuntu? We've usually just set the `os` to `linux` and used what Travis defaults to for this but there have been some cases where we've had to use a specific version (e.g. for Java 7 usage).

Reply:
No reason in particular other than it's equivalent to the latest version of ubuntu available in travis and to be more explicit in what we call out for the build environment. When we call out linux it looks like it should default to what Travis has set as the default linux image (The current default is `ubuntu 16.04`)